plugins/modules/notification/slack.py

```diff
@@ -187,33 +206,21 @@
 SLACK_INCOMING_WEBHOOK = 'https://hooks.slack.com/services/%s'
 SLACK_POSTMESSAGE_WEBAPI = 'https://slack.com/api/chat.postMessage'
 
-# Escaping quotes and apostrophes to avoid ending string prematurely in ansible call.
-# We do not escape other characters used as Slack metacharacters (e.g. &, <, >).
-escape_table = {
```

Comment:
Please remove escaping in another PR so it can be reverted in case it causes problems, without having to revert this feature as well.

Also, this is potentially breaking backwards compatibility, so I'm not sure you can simply drop this anyway.

Reply:
I've reverted this from this PR; will open a new one for it instead.